ios/Runner/RunnerRelease.entitlements

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>aps-environment</key>
+	<string>development</string>
```

Comment:
`RunnerRelease.entitlements`, `development` 🤔  Also both files look the same to me.

Reply:
These are auto-generated files.